pkg/services/hivelocity/device/device.go

```diff
@@ -441,6 +413,90 @@ func (s *Service) verifyAssociatedDevice(device *hv.BareMetalDevice) error {
 	return nil
 }
 
+// actionProvisionDevice re-provisions a device to remove it from cluster.
```

Comment:
Typo in comment: the name of the function is different.

Reply:
fixed